docs/source/installation.rst

```diff
@@ -3,43 +3,114 @@
 Installation
 ============
 
-sktime is available via PyPI using:
+``sktime`` requires an environment with python version 3.6 or higher.
+
+Release versions
+----------------
+
+For frequent issues with installation, consult the `Release versions - troubleshooting`_ section.
+
+Installing sktime from PyPI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``sktime`` releases are available via PyPI and can be installed via ``pip`` using:
 
 .. code-block:: bash
 
     pip install sktime
 
-But note that the package is actively being developed and currently not feature stable.
+This will install ``sktime`` with core dependencies, excluding interfaced packages.
 
+To install ``sktime`` with maximum dependencies, including all interfaced packages, install with the ``all_extras`` modifier:
 
-Development version
--------------------
-To install the latest development version of sktime, follow these steps:
+.. code-block:: bash
 
-1. Download the repository: :code:`git clone https://github.com/alan-turing-institute/sktime.git`
-2. Move into the root directory of the repository: :code:`cd sktime`
-3. Make sure you are on the main branch: :code:`git checkout main`
-4. Make sure your local version is up-to-date: :code:`git pull`
-5. Build package from source using: :code:`pip install --editable .` Please read below for more detailed instructions for specific operating systems.
+    pip install sktime[all_extras]
 
-Building sktime from source also requires
 
-- Cython >= 0.28.5 (available through :code:`pip install cython`)
-- OpenMP (see below for instructions)
+Installing sktime from conda
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. note::
+``sktime`` releases are available via ``conda`` from ``conda-forge``.
+They can be installed via ``conda`` using:
 
-   It is possible to build sktime without OpenMP support by setting the
-   ``SKTIME_NO_OPENMP`` environment variable (before cythonization). This is
-   not recommended since it will force some estimators to run in sequential
-   mode and their ``n_jobs`` parameter will be ignored.
+.. code-block:: bash
+
+    conda install -c conda-forge sktime
+
+This will install ``sktime`` with core dependencies, excluding interfaced packages.
+
+To install ``sktime`` with maximum dependencies, including all interfaced packages, install with the ``all_extras`` modifier:
+
+.. code-block:: bash
+
+    conda install -c conda-forge sktime[all_extras]
```

Comment:
it actually doesn't - thanks for pointing this out.
an installation starts (due to flexible solve), but `conda` handles package variants and extras differently, so this will not work.

This looks like the latest on the topic?
https://github.com/conda/conda/issues/7502

In any case, what should I write then: if you want all optional dependencies easily, use `pip`, not `conda`?

Reply:
Not sure, the requirements management isn't ideal at the moment. The `pip install sktime[extras]` syntax also isn't very widely known. It would be nice to have a single conda environment yaml file that people can install. Alternatively, you install the conda environment, activate it and then from inside that environment, use the pip install command.

Reply:
perhaps conda meta-packages are the way to go?

Reply:
anyway, fixed the reference